eksctl/2i2c-aws-us.jsonnet

```diff
@@ -42,6 +42,37 @@ local notebookNodes = [
     },
 ];
 
+local dedicatedNotebookNodes = [
```

Comment:
You can add these dirctly into `notebookNodes`, without needing a separate variable for them - exactly the same way as we add them to terraform.

Reply:
The contents don't need to change.

Reply:
Thanks @yuvipanda!

The `dedicatedNotebookNodes` list added a different prefix to the nodes' name as the node name is constructed via prefix + instance type and needed a way to give them unique ids.

So I've now added them to notebookNodes but with a `dedicated` suffix. Does that sound good?

Reply:
@GeorgianaElena (in lieu of Yuvi's time difference to help you move forward) this sounds excellent to me

Reply:
Thanks @sgibson91!